include/zephyr/bluetooth/l2cap.h

```diff
@@ -234,12 +234,115 @@ struct bt_l2cap_le_chan {
  */
 #define BT_L2CAP_LE_CHAN(_ch) CONTAINER_OF(_ch, struct bt_l2cap_le_chan, chan)
 
+/** L2CAP Endpoint Link Mode. Basic mode. */
+#define BT_L2CAP_BR_LINK_MODE_BASIC  0x00
+/** L2CAP Endpoint Link Mode. Restransmission mode. */
+#define BT_L2CAP_BR_LINK_MODE_RET    0x01
+/** L2CAP Endpoint Link Mode. Flow control mode. */
+#define BT_L2CAP_BR_LINK_MODE_FC     0x02
+/** L2CAP Endpoint Link Mode. Enhance restransmission mode. */
+#define BT_L2CAP_BR_LINK_MODE_ERET   0x03
+/** L2CAP Endpoint Link Mode. Streaming mode. */
+#define BT_L2CAP_BR_LINK_MODE_STREAM 0x04
+
+/** Frame Check Sequence type. No FCS. */
+#define BT_L2CAP_BR_FCS_NO			 0x00
+/** Frame Check Sequence type. 16-bit FCS. */
+#define BT_L2CAP_BR_FCS_16BIT		 0x01
+
 /** @brief BREDR L2CAP Endpoint structure. */
 struct bt_l2cap_br_endpoint {
 	/** Endpoint Channel Identifier (CID) */
 	uint16_t				cid;
 	/** Endpoint Maximum Transmission Unit */
 	uint16_t				mtu;
+#if defined(CONFIG_BT_L2CAP_RET) || defined(CONFIG_BT_L2CAP_FC) || \
+	defined(CONFIG_BT_L2CAP_ENH_RET) || defined(CONFIG_BT_L2CAP_STREAM)
+	/** Endpoint Link Mode.
+	 *  The value is defined as BT_L2CAP_BR_LINK_MODE_*
+	 */
+	uint8_t					mode;
```

Comment:
Same here and anywhere else if you did the same. Use just spaces for alignment and tabs for indentation (or to put it in another way: the only place for tabs is in the very beginning of a line - as long as you have some non-tab character on a line there should not be any more tabs anywhere later in the line)

Reply:
Note: I know there are inconsistencies wrt this in the Bluetooth code base, but we should try to move toward the convention that I just described.

Reply:
Yes. Updated.